CONTRIBUTING.md

```diff
@@ -15,7 +15,7 @@ Please note that this project is released with a [Contributor Code of Conduct][c
 0. Configure and install the dependencies: `bundle install`
 0. Make sure the tests pass on your machine: `bundle exec rspec spec`
 0. Create a new branch: `git checkout -b my-branch-name`
-0. Make your change, add tests, and make sure the tests still pass
+0. Make your change, add tests, and make sure the tests still pass and that no warnings are raised
```

Comment:
IIRC warnings go to STDERR, so you should be able to just enforce that STDERR is empty in CI